ports/remill/portfile.cmake

```diff
@@ -0,0 +1,37 @@
+vcpkg_from_github(
+  OUT_SOURCE_PATH SOURCE_PATH
+  REPO lifting-bits/remill
+  REF e7c0e3f9f7f482b6bcc336cd98b8afd4129c6e3b
+  SHA512 49ee8db4dbf097e89046f3fce3de85ea35703d329f44b3388af992664760044aa8a9e7d0dac1c71e914b0fe0293f5ec10cc8ef4de5e5a118a492b6b2581b9b8c
+  HEAD_REF vcpkg-manifest
```

Comment:
I think I mentioned this already privately, but ideally we use a commit on `master` branch before merging this